docs/tutorials/how-to-deploy-a-serverless-function.md

```diff
@@ -0,0 +1,208 @@
+  </TabItem>
+</Tabs>
+
+</details>
+
+## Deployment Guide
+
+## 1. Create the Genezio Configuration File
+
+Now, create a `genezio.yaml` file in the root directory of your project.
+
+This file will contain the configuration needed to deploy your backend using Genezio. Here is an example configuration.
+
+:::info
+
+1. You might need to replace the `entry` field with the name of your main application file.
+2. You might need to replace the `path` field with the path relative at **genezio.yaml** file.
+3. This example configuration works if **genezio.yaml** is in the same directory as your main application file.
+   :::
+
+```yaml title="genezio.yaml"
+# The name of the project.
+name: functions-app
+# The region where the project is deployed. Available regions: us-east-1, eu-central-1
+region: us-east-1
+# The version of the Genezio YAML configuration to parse.
+yamlVersion: 2
+backend:
+  # The root directory of the backend.
+  path: ./
+  # Information about the backend's programming language.
+  language:
+    # The name of the programming language.
+    name: js
+    # The package manager used by the backend.
+    packageManager: npm
+  # Information about the backend's functions.
+  functions:
+    # The name (label) of the function.
+    - name: hello-world-function
+      # The path to the function's code.
+      path: ./
+      # The name of the function handler
+      handler: handler
+      # The entry point for the function.
+      entry: app.mjs
+```
+
+This configuration file specifies the project name, deployment region, and details about the backend.
+
+## 4. Test Your App Locally
+
+Before deploying your app, you can test it locally to ensure it's working correctly.
+
+Run the following command in your terminal:
+
+```bash
+genezio local
+```
+
+In your terminal, you should be able to see all the local URLs of your functions. You can then test them in your browser or using a tool like Postman.
+
+## 5. Deploy your project
+
+Finally, deploy your project. A browser window will open, and you will be prompted to log in to your Genezio account and authorize the CLI to make the deployment.
+Run the following command in your terminal:
+
+```bash
+genezio deploy
+```
+
+If your application use environment variables, you can deploy them using the following command:
+
+```bash
+genezio deploy --env <path-to-your-env-file>
+```
+
+:::info
+You need to deploy your environment variables single time.
+After that, you can deploy your project without the `--env` flag.
+:::
+For more information about environment variables, you can check the [official documentation](/docs/project-structure/backend-envinronment-variables.md).
+
+## Test your app
+
+After deploying your application, you can test it to ensure it's running correctly. To verify that your functions are working, open a web browser and navigate to the URL provided for your deployed function.
+
+This URL can be found in the deployment output under the `Functions Deployed` section.
```

Comment:
I would remove this, I cannot find a "Function Deployed" section in the dashboard.